src/pages/PromptPage.vue

```diff
@@ -73,6 +73,24 @@ onMounted(async () => {
   comments.value = commentStore.getComments
 })
 
+async function onShare() {
+  const title = document.title
+  const url = document.querySelector('link[rel=canonical]') ? document.querySelector('link[rel=canonical]').href : document.location.href
+  const text = 'I find this usefully'
+  try {
+    await navigator.share({
+      title,
+      url,
+      text
+    })
```

Comment:
Hello @arnonrdp 
I'm using here native navigation function to be able to share. 
This function is only available in secure contexts

[the doc](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/share)

![image](https://user-images.githubusercontent.com/28951070/202517760-3e983000-e1e6-4fe0-a390-036c9f1d5878.png)

Reply:
Noted the need of https, it is alright.